src/main/java/edu/ucla/library/prl/harvester/handlers/AddInstitutionHandler.java

```diff
@@ -0,0 +1,47 @@
+
+package edu.ucla.library.prl.harvester.handlers;
+
+import org.apache.http.HttpStatus;
+
+import edu.ucla.library.prl.harvester.Institution;
+import edu.ucla.library.prl.harvester.InvalidInstitutionJsonException;
+import edu.ucla.library.prl.harvester.MediaType;
+import edu.ucla.library.prl.harvester.Op;
```

Comment:
Probably brain isn't firing on all cylinders right now, but... you import Op into all of the handler classes, but I don't see where you use it (other than a javadoc link). Is it really necessary? And if it is, why separately import it in each handler, rather than once in AbstractInstitutionRequestHandler?

Reply:
I don't think importing it in the abstract handler would work because it's used in the javadocs of the subclassed handler (that sort of thing would only work if you were inheriting a method with javadocs from the super class, I think).

I don't have an opinion on "is it really necessary?" I think the intention of the enum form is that they are "self-documenting" (the variable name looks like the value we use in the openapi spec). Is there value linking to the enum definition in the Javadocs? Right now, the javadocs don't have an explanation for each of the values, but are just grouping them:

```
    /**
     * A constant for the "get status" operation.
     */
    getStatus,

    /**
     * Institution operations.
     */
    addInstitution, getInstitution, listInstitutions, removeInstitution, updateInstitution
```

That doesn't seem to be adding much value to me. Btw, looks like the maven javadocs plugin needs configuration... currently no stylesheet is being used.

Reply:
I can see how it might be excessive given our naming convention of <OperationId>Handler. Will remove.